### ISSUE TITLE
az storage blob generate-sas requires --permissions now

### ISSUE THREAD

**Issue by wasker** [Azure/azure-cli#14138]
> ### `az feedback` auto-generates most of the information requested below, as of CLI version 2.0.62

**Describe the bug**

`az storage blob generate-sas` command started requiring `--permissions` flag and doesn't make it exclusive to `--policy-name`.

**To Reproduce**

Run `az storage blob generate-sas` with all required args, except `--permissions`, specify `--policy-name` instead. Alternatively, specify both `--permissions` and `--policy-name`.

If `--permissions` is not specified, the command fails on arg validation.

If both `--policy-name` and `--permissions` are specified, command succeeds, but SAS token cannot be used, because both permissions and policy name were used for generating the token, which is prohibited by Storage.

**Expected behavior**

Valid SAS token is generated using policy specified in `--policy-name`, `--permissions` arg is not required, if `--policy-name` was specified.

**Environment summary**

```
"azure-cli": "2.8.0",
"azure-cli-command-modules-nspkg": "2.0.3",
"azure-cli-core": "2.8.0",
"azure-cli-nspkg": "3.0.4",
"azure-cli-telemetry": "1.0.4",
"extensions": {
  "aks-preview": "0.4.53",
  "front-door": "1.0.7",
  "resource-graph": "1.1.0"
}
```

**Comment by yonzhan** [comment]
add to S173

**Comment by Juliehzl** [comment]
it is available in azure cli 2.9.1. please have a try.

**Comment by wasker** [comment]
Confirmed it works in 2.9.1. Thanks for quick turnaround!
